python/protobuf.c

```diff
@@ -302,6 +302,13 @@ const char* PyUpb_GetStrData(PyObject* obj) {
   }
 }
 
+const char* PyUpb_CheckStrData(PyObject* obj) {
```

Comment:
I usually avoid "Check" as a name since it can be confused with CHECK failing in the google3 context.  What about "Verify"?

Reply:
Done.